phalcon/http/response.zep

```diff
@@ -14,6 +14,7 @@
  +------------------------------------------------------------------------+
  | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
  |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ |          Zamrony P. Juhara <zamronypj@yahoo.com>                       |
```

Comment:
Why? I thought every contributors have right to put their name. I thought we value every contribution even smallest one? I do not get financial reward to do this work so putting my name is my reward. Your statement discourages people to contribute.